static/css/crateio-rtd.css

```diff
@@ -176,6 +180,8 @@ blockquote {
   padding-bottom: 10px;
   z-index: 11;
   width: 100%;
+  font-weight: 600;
+  min-width: 1007px;
```

Comment:
🤔
maybe let's make 2 lines instead of fixed width
what if I have a small screen?